conjure-java-undertow-runtime/src/main/java/com/palantir/conjure/java/undertow/runtime/ConjureBodySerDe.java

```diff
@@ -252,4 +249,23 @@ private static void safelyDrainRequestBody(HttpServerExchange exchange) {
             adapter.tag(target, "contentType", BINARY_CONTENT_TYPE);
         }
     }
+
+    /**
+     * Gets the request {@code Content-Type} header if exactly one value exists, otherwise throws
+     * a {@link SafeIllegalArgumentException}. This prevents unexpected behavior when multiple
```

Comment:
```suggestion
     * Gets the request {@code Content-Type} header if exactly one value exists, otherwise logs
     * a warning. This notifies us in the unexpected case when multiple
```